curriculum/challenges/english/25-front-end-development/quiz-asynchronous-javascript/66edd630f7666cfa54b404d0.md

```diff
@@ -17,439 +17,439 @@ Answer all of the questions below correctly to pass the quiz.
+What is the use of the `.finally()` method in a promise?
 
 #### --distractors--
 
-Placeholder distractor 1
+It returns the result of the promise.
 
 ---
 
-Placeholder distractor 2
+It stops the execution of subsequent promise handlers.
 
 ---
 
-Placeholder distractor 3
+It retries the promise if it fails.
 
 #### --answer--
 
-Placeholder answer
+It executes code after a promise is fulfilled or rejected, regardless of the outcome.
 
 ### --question--
 
 #### --text--
 
-Placeholder question
+What is the syntax for declaring an async function?
 
 #### --distractors--
 
-Placeholder distractor 1
+`function myFunction() {}`
 
 ---
 
-Placeholder distractor 2
+`function async myFunction() {}`
 
 ---
 
-Placeholder distractor 3
+`async {} => ()`
 
 #### --answer--
 
-Placeholder answer
+`async function myFunction() {}`
 
 ### --question--
 
 #### --text--
 
-Placeholder question
+What does the `await` keyword do in an async function?
 
 #### --distractors--
 
-Placeholder distractor 1
+It creates a new promise.
 
 ---
 
-Placeholder distractor 2
+It executes code immediately without waiting.
 
 ---
 
-Placeholder distractor 3
+It returns the result of the promise.
 
 #### --answer--
 
-Placeholder answer
+It pauses the execution of the function until the promise is resolved.
 
 ### --question--
 
 #### --text--
 
-Placeholder question
+What is the purpose of the Geolocation API?
 
 #### --distractors--
 
-Placeholder distractor 1
+To manipulate HTML elements.
 
 ---
 
-Placeholder distractor 2
+To create and manage map elements.
 
 ---
 
-Placeholder distractor 3
+To control the device's camera.
 
 #### --answer--
 
-Placeholder answer
+To retrieve the geographical location of a device.
 
 ### --question--
 
 #### --text--
 
-Placeholder question
+Which method of the Geolocation API is used to get the current position of the device?
 
 #### --distractors--
 
-Placeholder distractor 1
+`getLocation()`
 
 ---
 
-Placeholder distractor 2
+`getPosition()`
 
 ---
 
-Placeholder distractor 3
+`getLatLong()`
 
 #### --answer--
 
-Placeholder answer
+`getCurrentPosition()`
 
 ### --question--
 
 #### --text--
 
-Placeholder question
+What parameters can be passed to the `getCurrentPosition()` method?
 
 #### --distractors--
 
-Placeholder distractor 1
+Only the success callback function.
 
 ---
 
-Placeholder distractor 2
+Only options.
 
 ---
 
-Placeholder distractor 3
+One the error callback function.
 
 #### --answer--
 
-Placeholder answer
+Success and error callback functions, along with options.
 
 ### --question--
 
 #### --text--
 
-Placeholder question
+What will the following code return? `fetch('https://api.example.com/data')`
 
 #### --distractors--
 
-Placeholder distractor 1
+The JSON response directly.
 
 ---
 
-Placeholder distractor 2
+An error if the URL is not reachable.
 
 ---
 
-Placeholder distractor 3
+A synchronous response.
 
 #### --answer--
 
-Placeholder answer
+A Promise that resolves to a Response object
```

Comment:
```suggestion
A `Promise` that resolves to a `Response` object
```
Since these refer in example to a concrete promise and response objects, maybe they should be backticked...